samza-api/src/main/java/org/apache/samza/task/TaskContext.java

```diff
@@ -58,10 +58,9 @@
   /**
    * Method to allow user to return customized context
```

Comment:
Will do. Only mentioned this here since TaskContext isn't part of the operators package so will be easy to miss.